### DISCUSSION TITLE
Moving between split

### DISCUSSION THREAD

**Discussion by tuanbass** [neovim/neovim#25173] [Q&A] [answered]
Hi friends, 

Is there any way to "remember" the split that I jump from? 
Assume I have vim/nvim open with 3 splits, as the pictures. And I'm in **top-right** split
`C-w h` => Now I'm in **left** split. 
`C-w l` to go **right**.

Now  , my focused split depend on the position of cursor in `left` one.
If cursor at line  1-12, I will be in `top right`. Else I will be in `bottom right`
Is there any way to change the behavior, so I can go to exact the split that I left it before (**top right**)? 

![image](https://github.com/neovim/neovim/assets/9334023/5ca1e8c5-2997-46a0-814f-239a2416bd05)

**Comment by gegoune** [comment, marked as answer]
`<C-w>p`
